mfs limitation ftp remote

edwardxml opened the pull request:
Limitation consistent with `rclone about remote:` and mentioned in the rclone forum

#### What is the purpose of this change?

Reflect issue from forum in rclone documentation

#### Was the change discussed in an issue or in the forum before?

https://forum.rclone.org/t/rclone-union-mfs-most-free-space-not-working-for-ftp/20346
#### Checklist

- [ x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] I have added tests for all changes in this PR if appropriate.
- [ ] I have added documentation for the changes if appropriate.
- [ ] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x ] I'm done, this Pull Request is ready for review :-)

ncw commented on docs/content/ftp.md:
```diff
@@ -236,6 +236,9 @@ through rclone are time of upload.
 Rclone's FTP backend does not support any checksums but can compare
 file sizes.
 
+When an FTP remote is a member of an rclone union remote `mfs`
```
That should probably say

> When a remote which doesn't support [rclone about](https://rclone.org/overview/#optional-features) (eg FTP, Box, S3 etc) is a member...

ncw reviewed (commented):
Comment inline :-)

Thank you!

edwardxml commented:
Modified ftp.md to make the more general points re 'about' capability. Had been foxed by the OP on  https://forum.rclone.org/t/rclone-union-mfs-most-free-space-not-working-for-ftp/20346 suggesting rclone about worked.

Also reflected same changes in overview.md and removed repeated 'some'.

ncw commented:
That looks great now - will merge - thank you :-)
